almost_tcp/packet_fixtures.py

```diff
@@ -43,7 +77,10 @@ async def collector(ctx):
                 if ready == 1 and valid == 1:
                     # We just transferred a payload byte.
                     self.body = self.body + bytes([payload])
-                ready = self.is_ready()
+                    ready = self.is_ready()
+                else:
+                    # Maybe become ready, don't become un-ready.
+                    ready = ready | self.is_ready()
```

Comment:
Misread this at first, maybe rephrase the comment: "don't become un-ready until we transfer a payload byte"?

Reply:
Done in 17db2ef (here and elsewhere)